doc/manual/src/command-ref/nix-env.md

```diff
@@ -238,31 +238,37 @@ a number of possible ways:
 
 ## Examples
 
-To install a specific version of `gcc` from the active Nix expression:
+To install a package using a specific attribute path from the active Nix expression:
+
+```console
+$ nix-env -i -A gcc40mips
+installing `gcc-4.0.2'
+$ nix-env -i -A xorg.xorgserver
```

Comment:
```suggestion
$ nix-env -iA gcc40mips
installing `gcc-4.0.2'
$ nix-env -iA xorg.xorgserver
```
Everywhere else this is used. Not sure what is better for the doc, I let you decide.

Reply:
Yeah, I did not change the original format when moving it around but I guess it is a good idea to unify this when touching it. Examples should show what people want to use, after all.